Packs/MicrosoftGraphMail/pack_metadata.json

```diff
@@ -2,7 +2,7 @@
     "name": "Microsoft Graph Mail",
     "description": "Microsoft Graph lets your app get authorized access to a user's Outlook mail data in a personal or organization account.",
     "support": "xsoar",
-    "currentVersion": "1.0.8",
+    "currentVersion": "1.1.0",
```

Comment:
```suggestion
    "currentVersion": "1.0.9",
```

Reply:
@daryakoval When making this change, you will need to rename the release notes file to `1_0_9.md` as well.